src/components/RankingList.tsx

```diff
@@ -0,0 +1,119 @@
+import { rankingItemMock } from "@/assets/rankingItemMock";
+import styled from "@emotion/styled";
+import Divider from "@/components/Divider";
+import { useState } from "react";
+import type { ThemeType } from "@/types/ThemeType";
+import { useTheme } from "@emotion/react";
+import Button from "@/components/common/Button";
+
+const RANKING_LIST_ITEM_VIEW_COUNT = 6;
+
+const RankingList = () => {
+  const [viewCount, setViewCount] = useState(RANKING_LIST_ITEM_VIEW_COUNT);
+  const theme = useTheme();
+  const isClosed = () => {
+    return viewCount === RANKING_LIST_ITEM_VIEW_COUNT;
+  };
```

Comment:
```suggestion
  const closed = viewCount === RANKING_LIST_ITEM_VIEW_COUNT
```
지금은 함수 대신 값으로 표현해서 사용해도 무방해보이네요!